.editorconfig

```diff
@@ -18,6 +17,7 @@ indent_style = space
 indent_size = 2
 
 [*.hbs]
+insert_final_newline = false
 indent_style = space
 indent_size = 2
 
```

Comment:
This file does not have the same "insert_final_newline" entry occurrences as the other repos.